### DISCUSSION TITLE
ensure does not work on v2.0.0-5

### DISCUSSION THREAD

**Discussion by vibha-somashekhara** [artilleryio/artillery#1250] [Artillery]
I have seen that v2.0 is still under development. As I was using v2.0.0-5, I found that adding ensure in my test scripts did not fail the test when condition was not met. Ensure condition was simply not checked and I did not see anything after summary report.

I changed it to v1.7.9 and now I can see ensure conditions are being checked.
`ensure condition failed: ensure.p99 < 0`

**Comment by hassy** [comment]
hi @vibha-somashekhara 👋  - the functionality has been moved into a dedicated plugin: https://github.com/artilleryio/artillery-plugin-ensure

thank you for flagging this up, we'll update the docs
